--- org.sat4j.pb/META-INF/MANIFEST.MF.sav	2010-03-26 15:20:38.390630122 +0200
+++ org.sat4j.pb/META-INF/MANIFEST.MF	2010-03-26 15:20:50.549378523 +0200
@@ -21,5 +21,4 @@ Specification-Vendor: Daniel Le Berre
 Implementation-Title: SAT4J
 Implementation-Version: 9.9.9.token 
 Implementation-Vendor: CRIL CNRS UMR 8188 - Universite d'Artois
-Class-Path: org.sat4j.core.jar
 Bundle-RequiredExecutionEnvironment: J2SE-1.4
